README.rst

```diff
@@ -1,28 +1,68 @@
 |Build Status| |Coveralls| |npm_version| |npm_downloads| |license|
 
 frontend-app-payment
-=========================
+====================
 
 Please tag **@edx/arch-team** on any PRs or issues.  Thanks.
 
 Introduction
 ------------
 
-React app for payment.
+Micro-frontend for the single-page payment/checkout process.
 
-Important Note
---------------
+Getting Started
+---------------
 
-The production Webpack configuration for this repo uses `Purgecss <https://www.purgecss.com/>`_ 
+1. Clone this repo:
+
+  ``git clone https://github.com/edx/frontend-app-payment.git``
+
+2. Install npm dependencies:
+
+  ``cd frontend-app-payment && npm install``
+
+3. Start the dev server:
+
+  ``npm start``
```

Comment:
Should we link to devstack and mention that it's a prereq?

Reply:
... right. :)